### ISSUE TITLE
com.azure.authenticator no notifications

### ISSUE THREAD

**Issue by FauconSpartiate** [AdguardTeam/AdguardForAndroid#4990]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

V4.1

### Environment

```markdown
- OS: Android 13
- Device: Oneplus 8T
- Firmware: KB2003_13.1.0.581(EX01)
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

AdGuard Base filter

### Issue Details

Steps to reproduce:
1. Open app
2. On another device, trigger a 2FA login request

### Expected Behavior

Notification pops up on Android device, when app is in background and in foreground

### Actual Behavior

Notification never comes, neither in background, nor foreground

### Additional Information

Originally from https://github.com/AdguardTeam/AdguardFilters/issues/160972 and https://github.com/AdguardTeam/AdguardFilters/issues/160978

**Comment by FauconSpartiate** [comment]
Nope, I narrowed it down the the base filter.

**Comment by FauconSpartiate** [comment]
I just tested it again to send the logs, and I noticed that the issue probably isn't AdGuard. Even with filtering completely turned off, the first 2FA request always fails regardless.
My personal account shows the notification on the first try, while my work account only does it on the second one.

Very weird bug, I'll try to figure out what happens there, and I'll reopen this issue if I find evidence that it's because of AdGuard.

Sorry for bothering
